src/decorators/dispatch.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  NgRedux,
+} from '../components/ng-redux';
+
+
+export function dispatch(options?: any) {
```

Comment:
I was playing around with an idea to be able to provide your own custom dispatcher if you wanted instead of needing to rely on the singleton instance of NgRedux. 

I'll probably just drop this for now, and explore adding it back later as it's not serving any purpose right now, and rather get the basic use case working well first.